diff.py

```diff
@@ -1961,7 +1963,29 @@ def process(dump: str, config: Config) -> List[Line]:
             row_parts = [part.lstrip() for part in row.split(" ", 1)]
         mnemonic = row_parts[0].strip()
         args = row_parts[1] if len(row_parts) >= 2 else ""
-        row = mnemonic + "\t" + args.replace("\t", "  ")
+
+        if arch.name == "ppc":
```

Comment:
this logic should go on a callback on AsmProcessorPPC

Reply:
this resulted in some more refactoring, note the function pre_process gets called for all archs, and the default behavior is 

>row = mnemonic + "\t" + args.replace("\t", "  ")

but extra steps for ppc